LICENSE

```diff
@@ -1,7 +1,7 @@
                     GNU GENERAL PUBLIC LICENSE
                        Version 3, 29 June 2007
 
- Copyright (C) 2007 Free Software Foundation, Inc. <https://fsf.org/>
+ Copyright (C) 2022 krazykirby99999
```

Comment:
You shouldn't change this. That's the copyright notice for the license document itself :)

Just put it next to the SPDX header instead.